bedrock/firefox/views.py

```diff
@@ -594,7 +600,12 @@ def get_template_names(self):
                 if country == "GB" or locale == "en-GB":
                     template = "firefox/whatsnew/whatsnew-fx112-eu-privacy.uk.html"
                 else:
-                    template = "firefox/whatsnew/index.html"
+                    if variant == "2":
+                        template = "firefox/whatsnew/whatsnew-fx112-en-bundle.html"
```

Comment:
I've added both a geo check for the TC script so people outside US/CA shouldn't be enrolled, and also added a geo check in the view so people outside US/CA shouldn't even be able to view the template directly, just in case someone links to it. Since you're in the UK you should no longer be able to see the bundle page unless you add `&geo=us`